lib/puppet/pops/parser/egrammar.ra

```diff
@@ -857,8 +857,8 @@ heredoc
   : HEREDOC sublocated_text  { result = Factory.HEREDOC(val[0][:value], val[1]); loc result, val[0] }
 
 sublocated_text
-  : SUBLOCATE string    { result = Factory.SUBLOCATE(val[0], val[1]); loc result, val[0] }
-  | SUBLOCATE dq_string { result = Factory.SUBLOCATE(val[0], val[1]); loc result, val[0] }
+  : SUBLOCATE string    { result = Factory.SUBLOCATE(val[0], val[1]); }
+  | SUBLOCATE dq_string { result = Factory.SUBLOCATE(val[0], val[1]); }
```

Comment:
This change is expected - it has to do with the changes for heredoc parsing. This is the change in the grammar that make it necessary to also run the rake task to generate the parser.

Reply:
@jhelwig The resulting change in the grammar after conflict resolution looks correct. Then a regenerate parser was needed - if tests run green that must have been done the right way.